wp-content/themes/pub/wporg-learn-2024/src/course-outline/index.js

```diff
@@ -1,37 +1,33 @@
-/* global wporgCourseOutlineData, wporgCourseOutlineL10n */
+/* global wporgCourseOutlineL10n */
 
 import { Icon, drafts, lockOutline } from '@wordpress/icons';
 import { renderToString } from '@wordpress/element';
 
 document.addEventListener( 'DOMContentLoaded', () => {
-	if ( ! wporgCourseOutlineData ) {
-		return;
-	}
-
-	wporgCourseOutlineData[ 'in-progress' ]?.forEach( ( title ) => {
-		const lessonLinks = document.querySelectorAll( '.wp-block-sensei-lms-course-outline-lesson' );
-		lessonLinks.forEach( ( link ) => {
-			const span = link.querySelector( 'span' );
-			if ( span && span.textContent.trim() === title ) {
-				const statusIcon = link.querySelector( '.wp-block-sensei-lms-course-outline-lesson__status' );
-				if ( statusIcon ) {
-					statusIcon.outerHTML = renderToString(
-						<>
-							<Icon icon={ drafts } style={ { transform: 'scale(1.5)' } } />
-							<span className="screen-reader-text">{ wporgCourseOutlineL10n.inProgress }</span>
-						</>
-					);
-				}
-			}
-		} );
+	document.querySelectorAll( '.wp-block-sensei-lms-course-outline-lesson.is-in-progress' ).forEach( ( link ) => {
```

Comment:
Can you add some comments here? [This](https://github.com/WordPress/Learn/pull/2776/files#diff-699f649e8f6d71907216b7f3caf87fa82059948b488f7216ac4c30ff222321e5R36-R39) is what I tried to add in another PR. Thanks!

Reply:
Sure, how's this? https://github.com/WordPress/Learn/pull/2786/commits/830a321e8b6276a2b74f33144cdd2e2abbe1d501

Reply:
Looks good! 100x better than mine.